examples/ext_authz/docker-compose.yaml

```diff
@@ -33,7 +33,7 @@ services:
       - envoymesh
 
   ext_authz-opa-service:
-    image: openpolicyagent/opa:0.21.0-istio
+    image: openpolicyagent/opa:0.24.0-istio
```

Comment:
I'm not sure about this change; I couldn't get the OPA test to pass locally, is the OPA Envoy support v2 only? CC @lizan